python/ray/tune/tests/test_integration_wandb.py

```diff
@@ -1,15 +1,18 @@
 import os
 import tempfile
 from collections import namedtuple
+from dataclasses import dataclass
 from multiprocessing import Queue
-import unittest
+import pytest
```

Comment:
Nit: `pytest` should be grouped with `numpy`.

Reply:
done